include/emp/Evolve/Systematics.hpp

```diff
@@ -221,20 +250,21 @@ namespace emp {
     /// Remove an organism from this Taxon (after it dies).
     /// Removals must return true if the taxon needs to continue; false if it should deactivate.
     bool RemoveOrg() {
-      emp_assert(num_orgs > 0, num_orgs);
+      emp_optional_throw(num_orgs > 0, "Removing org from extinct taxon");
```

Comment:
A good way to do this might be adding a notice/link to the header comment for this file that it is being used to power PhylotrackPy and describe any considerations relevant to that

Reply:
Good idea! Done.